libkbfs/md_journal.go

```diff
@@ -780,6 +928,21 @@ func (j *mdJournal) put(
 		}
 	}()
 
+	if extra == nil {
+		// TODO: This could fail if the key bundle isn't part
```

Comment:
@aalness re. our discussion about `nil` `ExtraMetadata`s, this seems to indicate a need for keeping `extra` around with new versions, plus a flag to indicate whether it should be uploaded to the server.